expression/builtin_string_vec_test.go

```diff
@@ -106,6 +106,42 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 			childrenTypes: []types.EvalType{types.ETString, types.ETString, types.ETInt},
 			geners:        []dataGenerator{&randLenStrGener{1, 2}, &randLenStrGener{0, 10}, &rangeInt64Gener{0, 8}},
 		},
+		{
```

Comment:
Maybe the all results are 0, because str1 is not in str2 in general when they are random string. 
Please add some special test.

Reply:
Actually I see the `locate` test cases that already exist also use `randLenStrGener`. To avoid zero output there are cases whose substring is short, i.e. `&randLenStrGener{1, 2}`. Do you mean adding some constant cases or designing a new generator?

Reply:
Adding some constant cases. We can use selectStringGener to get the string we want.

Reply:
I have added some test cases. Please see if they meet our needs.